source/_patterns/basics/01-buttons/icon-button.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Icon Button
+---
+## Overview
+
+A button containing an icon is used as part of the [Patternfly toolbar](https://www.patternfly.org/pattern-library/forms-and-controls/toolbar/#_design) and in other places.
```

Comment:
This comment makes a lot of sense, but patternlab already tells you where each component is used. Do we need to add it again?